client/domain/PromptObjectFactory.js

```diff
@@ -59,6 +59,10 @@ tie.factory('PromptObjectFactory', [
       return this._mainFunction;
     };
 
+    Prompt.prototype.getOutputFunctionName = function() {
+      return this._outputFunction;
```

Comment:
@eyurko -- I'm wondering, shall we change outputFunction to outputFunctionName in the schema? I think it'd be good to distinguish clearly between the function and the name of the function, but would appreciate your thoughts on this.